index.js

```diff
@@ -1,4 +1,4 @@
-'use babel'
+'use babel';
```

Comment:
must not have semicolon after `'use babel'`. Check other files.

Reply:
But due to not having semicolon..in travis build was getting failed.

Reply:
No. It was generating warnings all the time.

Reply:
ok, revrting the semicolon.